### PR TITLE
Added suggestions button and translation (#3714)

### LINKED ISSUE #3714: Add the suggestions board link somewhere in the game
Thrive itself should have a button to "suggest a feature" that opens https://suggestions.revolutionarygamesstudio.com when clicked

Note that when https://github.com/Revolutionary-Games/Thrive/pull/3697 is done, this'll be an easy add to the other revolutionary games links, but I think additionally we should have a button maybe in the extras or options menu as well for this.

### PR DESCRIPTION
**Pull request by TheHungryBandito**
+ Added suggestions button with link to suggestions board.
+ Localization for the word "Suggestions"
+ English translation confirmed and fuzzy removed

**Brief Description of What This PR Does**

This PR adds a button to the 'Extras' menu containing a link to the suggestions board as well as the automated translations for "Suggestions".

**Related Issues**
closes #3714 

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
It seems you haven't pushed any changes to this branch, or you reverted them all?

**Comment by TheHungryBandito** [comment]
I reverted them after I gave it a double check and noticed that the "QUESTION" translation key was being removed. I will fix the issue I'm having and push some changes soon. I'll mark it as non-draft when I push them.

File: locale/en.po [hunk 1]
```diff
@@ -2167,19 +2167,23 @@ msgstr "Art Gallery"
 msgid "LICENSES"
 msgstr "Licenses"
 
-#: ../src/general/MainMenu.tscn:496
+#: ../src/general/MainMenu.tscn:420
+msgid "SUGGESTIONS"
```
Comment (hhyyrylainen) [inline locale/en.po:2171]: The key should be "VISIT_SUGGESTIONS_SITE" and the English text "Suggest a Feature"

File: locale/en.po [hunk 2]
```diff
@@ -2167,19 +2167,23 @@ msgstr "Art Gallery"
 msgid "LICENSES"
 msgstr "Licenses"
 
-#: ../src/general/MainMenu.tscn:496
+#: ../src/general/MainMenu.tscn:420
+msgid "SUGGEST_A_FEATURE"
+msgstr "Suggest A Feature"
```
Comment (hhyyrylainen) [inline locale/en.po:2172]: A native speaker can correct me, but I think usually in titles small words like "a" are *not* capitalized. Also small words are not necessary to put in the translation keys, often their content is changed entirely. That's why I think the key should be visit suggestions site like I said before as then the English text can be replaced if we feel like it with something completely different.

Comment (TheHungryBandito) [reply locale/en.po:2172]: Sorry, I misread the last comment and only just noticed my mistake. Fixing it now

File: src/general/MainMenu.tscn [hunk 3]
```diff
@@ -360,7 +360,6 @@ size_flags_vertical = 0
 text = "BACK"
 
 [node name="Extras" type="VBoxContainer" parent="MenuContainers/Menus/MarginContainer" groups=["MenuItem"]]
-visible = false
```
Comment (hhyyrylainen) [inline src/general/MainMenu.tscn:363]: Unintentional change?

**Review by hhyyrylainen — APPROVED** [review]
Fixed the one remaining problem with the extras menu being visible by default and solved merge conflicts.

**Pull request merged**